client/js/components/CardEventSmall/CardEventSmall.js

```diff
@@ -11,11 +11,14 @@ import {calculateRatingScore} from '../../lib/calculateRatingScore';
 import styles from './styles';
 import PropTypes from 'prop-types';
 
-const CardEventSmall = ({eventInfo, index}) => {
+const CardEventSmall = ({eventInfo, index, navigation}) => {
   const {category, title, image, price, reviews} = eventInfo;
 
   return (
     <TouchableOpacity
+      onPress={e => {
```

Comment:
what is `e`? Do you need the javascript event here? If not then just use `()` instead of `(e)`.